tests/fixtures/static_html/one_map.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>A page containing 1 map</title>
-    <script src="https://europa.eu/webtools/load.js" async></script>
+    <script src="https://webtools.europa.eu/load.js" async></script>
```

Comment:
The Webtools Smartloader is usually loaded with `defer`.
See 
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Important+guidelines+for+developers+and+implementers
* https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Widgets